components/src/atoms/list/List.scss

```diff
@@ -1,9 +1,8 @@
-@import '../../../assets/styles/_globals.scss';
+@import '../../../assets/styles/globals';
 
 $list__style: none !default;
 $list__item-padding: 5px 0 !default;
 $list__content-padding: 10px !default;
-$list__label-padding: 0 5px 5px 0 !default;
 $list__icon-color: $white !default;
```

Comment:
Please update alignment of variables in this file, you can use [this extension](https://marketplace.visualstudio.com/itemdetails?itemName=annsk.alignment)